### ISSUE TITLE
Mention where to find public keys to use with .sig files

### ISSUE THREAD

**Issue by jidanni** [Bitcoin-com/Wallet#44]
On https://github.com/Bitcoin-com/Wallet/releases
there are .sig files, but one can Google for hours and
still not find the public keys required to verify them.

**Comment by jidanni** [comment]
Indeed https://www.bitcoin.com/guides/your-guide-to-avoiding-bitcoin-fraud says
```To avoid scams and to be sure you are getting a real Bitcoin wallet, visit our wallet portal on Bitcoin.com or directly
download Bitcoin.com’s official wallet.
```
but nowhere anything about how to verify the signatures provided.

**Comment by nghiacc** [comment]
Me too, could not be able to download the public key using this command: gpg --recv-key BFFE82CAADC3F980
any hint is appreciated.

**Comment by magmahindenburg** [comment]
You can find the key here: http://pgp.mit.edu/pks/lookup?op=get&search=0xBFFE82CAADC3F980

**Comment by mihaifm** [comment]
I agree, this should be documented...currently the .sig file is pretty useless without some information on how to use it for verification.

**Comment by bitcartel** [comment]
The last release signed with key BFFE82CAADC3F980 was 4.10.1: https://github.com/Bitcoin-com/Wallet/releases/tag/4.10.1.

Subsequent releases have used different keys e.g. 5.1-rc5 signed with key 2C6D99B5 and most recently 5.2-rc1 signed with 2FC1D3E6.

**Comment by JulianTosh** [comment]
Chiming in...this needs to be resolved. Please publish an official signing key.

**Comment by JulianTosh** [comment]
Found it (the current key signing 5.2-hotfix2).

http://keys.gnupg.net/pks/lookup?search=0x2c6d99b5&fingerprint=on&op=index
